editor/animation_bezier_editor.cpp

```diff
@@ -29,6 +29,7 @@
 /*************************************************************************/
 
 #include "animation_bezier_editor.h"
+#include "editor/editor_node.h"
```

Comment:
Please always leave an empty line between the .cpp's own header (`animation_bezier_editor.h`) and the other Godot headers.